vsintegration/src/FSharp.Editor/LanguageService/WorkspaceExtensions.fs

```diff
@@ -247,3 +246,20 @@ type Project with
                 do! doc.FindFSharpReferencesAsync(symbol, (fun textSpan range -> onFound doc textSpan range), userOpName)
                     |> RoslynHelpers.StartAsyncAsTask ct
     }
+    
+    member this.GetFSharpCompilationOptionsAsync(ct: CancellationToken) =
```

Comment:
Can you update `Document.GetFSharpCompilationOptionsAsync` to use this one?

Reply:
They're not really the same, the one in Document is async, this one is background task.
We should migrate to task-based in a separate PR